src/lib/entities.ts

```diff
@@ -57,7 +55,6 @@ export class EntitiesOperator {
    */
   async getProfileEntity(address: string): Promise<ProfileEntity> {
     const entities: Entity[] = await this.catalystClient.fetchEntitiesByPointers(
-      EntityType.PROFILE,
```

Comment:
is this property no longer required, or is the method `fetchEntitiesByPointers` getting the `type` as `EntityType.PROFILE` by default?

Reply:
the new version of catalystClient removed that prop because wasn't necessary at all, all entities have unique ids, no matter the type